OOPConcepts/src/com/temperature/Temperature.java

```diff
@@ -0,0 +1,44 @@
+package com.temperature;
+
+public class Temperature {
+
+	private double number;
+
+	private TempScale scale;
+
+	public Temperature() {
+		number = 0.0;
+		scale = TempScale.FAHRENHEIT;
+	}
+
+	public Temperature(double number) {
+		this.number = number;
+		scale = TempScale.FAHRENHEIT;
+	}
+
+	public Temperature(TempScale scale) {
+		number = 0.0;
+		this.scale = scale;

```

Comment:
why is `this.scale = scale` repeated on this line, line 26 and line 38?

Reply:
I used this.scale to refer to the instance variable private TempScale scale since the parameterized variable's name is scale. Would you write it differently?

Reply:
actually no but I was kind of confused about the naming convention.